packages/comscore-analytics/types.ts

```diff
@@ -14,20 +14,51 @@ declare global {
   }
 }
 
-interface ComscoreAnalytics extends Analytics {
+/**
+ * `@frontity/comscore-analytics` package.
+ */
+interface ComscoreAnalytics extends Package {
   roots: Analytics["roots"] & {
     comscoreAnalytics: ReactType;
   };
   state: Analytics["state"] & {
+    /**
+     * State properties for the Comscore analytics package.
+     */
     comscoreAnalytics: {
+      /**
+       * Array of tracking ids associated to a Comscore client.
+       */
       trackingIds: string[];
```

Comment:
Shouldn't Comscore support a singular `trackingId` to be consistent with Google Analytics and GTM

Reply:
Ok, I'll do that.